src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerLimits.cs

```diff
@@ -170,13 +152,51 @@ public TimeSpan KeepAliveTimeout
         /// </remarks>
         public TimeSpan RequestHeadersTimeout
         {
-            get
+            get => _requestHeadersTimeout;
+            set => _requestHeadersTimeout = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the maximum number of connections that will accept requests. When set to null, the number of connections is unlimited.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to <see cref="uint.MaxValue"/>.
+        /// </remarks>
+        public uint? MaxConcurrentConnections
```

Comment:
While I don't think we're still worried about  CLS compliance these days, let's avoid unsigned numeric types for consistency with the other Kestrel limits and other .NET APIs in general.

Can we just make it a long and default to null?